app/redux/home/HomeSlice.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    ActionReducerMapBuilder,
+    Draft,
+    PayloadAction,
+    createSlice,
+} from '@reduxjs/toolkit';
+import { UserSchemaType, getUsersThunk } from '../../services';
+
+interface InitialStateType {
+  users: UserSchemaType[] | undefined;
+  page: number;
+  isLoading: boolean;
+  error: string;
+}
+
+const initialState: InitialStateType = {
+  users: undefined,
+  page: 1,
```

Comment:
Don't use static page number in slice

Reply:
it is initial value of state, any how we should provide any initial number!